keyboards/rart/rart75hs/readme.md

```diff
@@ -0,0 +1,30 @@
+# [RART 75 Hotswap](https://github.com/alabahuy/RART/tree/master/RART75)
+
+![RART 75](https://user-images.githubusercontent.com/30220306/157791657-8c7f9093-d42f-41f1-8252-81fbb21cb080.jpg)
```

Comment:
Please rehost this on an image hosting site, such as Imgur.

Reply:
done :) @drashna